External/FEXCore/Source/Utils/ELFLoader.cpp

```diff
@@ -412,7 +412,7 @@ void ELFContainer::CalculateSymbols() {
       uint64_t offset = SymTabHeader->sh_offset + i * SymTabHeader->sh_entsize;
       Elf32_Sym const *Symbol =
           reinterpret_cast<Elf32_Sym const *>(&RawFile.at(offset));
-      if (ELF32_ST_VISIBILITY(Symbol->st_other) != STV_HIDDEN &&
+      if (Symbol->st_other & 0x3 != STV_HIDDEN &&
```

Comment:
Why was the macro expanded here? It's much cleaner to have it wrapped in a macro

Reply:
The macro doesn't exist on android's version of `elf.h`

Looks like it's a GNU libc extension to the elf.h library.

Reply:
Might be nice to define our own macro or lambda for this then :)

Reply:
Also, this is wrong. Need brackets around `Symbol->st_other & 0x3`

Reply:
Ok, I'm now redefining the macro if it's not defined.